### ISSUE TITLE
unclear note description for registering

### ISSUE THREAD

**Issue by clement559** [clement559/pe#14]
Whilst I understand that 2 patients may have the same name, it is not clear as to what it means for the same username with 2 or more passwords access different accounts. This may be read as any users with additional passwords can choose which account they would like to access with any of the passwords. Perhaps it would be better to rephrase this to avoid any confusion.

![image.png](https://raw.githubusercontent.com/clement559/pe/main/files/5c87520b-494f-45b0-a1a2-fd6a1c8d4c48.png)

**Comment by nus-pe-script** [comment]
Your response not required for this bug as the team has accepted the bug as it is.
